package.json

```diff
@@ -60,6 +60,7 @@
   },
   "dependencies": {
     "@jridgewell/trace-mapping": "^0.3.18",
+    "get_cpus_length": "^1.0.3",
```

Comment:
I don't think it is a good idea to read system files https://github.com/xiaoxiaojx/get_cpus_length/blob/master/index.js#L21

Reply:
In our testing, all Linux containers have this issue.

Currently, no better method has been found. If this method is not used, would you consider not setting the concurrency quantity completely according to the CPU cores by default?

For example: `Math.min(4, os.cpus().length)`

I think this fix is very important for robustness, especially for large projects.

Reply:
> I think this fix is very important for robustness, especially for large projects.

I agree with you, just want to find the better solution. How we can detect is it a `docker container`? Maybe we can disable/improve logic without reading system files.